src/main/java/frc/robot/commands/auto/Autos.java

```diff
@@ -0,0 +1,117 @@
+package frc.robot.commands.auto;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import frc.lib.utils.AllianceFlipUtil;
+import frc.robot.FieldConstants;
+import frc.robot.Orchestrator;
+import frc.robot.subsystems.drive.Drive;
+import org.littletonrobotics.junction.Logger;
+
+public class Autos {
+  private final Drive drive;
+  private final Orchestrator orchestrator;
+  private Translation2d noteTranslation;
+  private Translation2d secondNoteTranslation;
+  private Translation2d thirdNoteTranslation;
+
+  public Autos(Drive drive, Orchestrator orchestrator) {
+    this.drive = drive;
+    this.orchestrator = orchestrator;
+  }
+
+  public enum StartingPosition {
+    LEFT,
+    CENTER,
+    RIGHT
+  }
+
+  private void setNotePositions(StartingPosition position) {
+    switch (position) {
+      case LEFT -> {
+        this.noteTranslation =
+            AllianceFlipUtil.apply(FieldConstants.StagingLocations.spikeTranslations[0]);
+        this.secondNoteTranslation =
+            AllianceFlipUtil.apply(FieldConstants.StagingLocations.spikeTranslations[1]);
+        this.thirdNoteTranslation =
+            AllianceFlipUtil.apply(FieldConstants.StagingLocations.spikeTranslations[2]);
+      }
+      case CENTER -> {
+        this.noteTranslation =
+            AllianceFlipUtil.apply(FieldConstants.StagingLocations.spikeTranslations[1]);
+        this.secondNoteTranslation =
+            AllianceFlipUtil.apply(FieldConstants.StagingLocations.spikeTranslations[2]);
+        this.thirdNoteTranslation =
+            AllianceFlipUtil.apply(FieldConstants.StagingLocations.spikeTranslations[0]);
+      }
+      case RIGHT -> {
+        this.noteTranslation =
+            AllianceFlipUtil.apply(FieldConstants.StagingLocations.spikeTranslations[2]);
+        this.secondNoteTranslation =
+            AllianceFlipUtil.apply(FieldConstants.StagingLocations.spikeTranslations[1]);
+        this.thirdNoteTranslation =
+            AllianceFlipUtil.apply(FieldConstants.StagingLocations.spikeTranslations[0]);
+      }
+    }
+
+    Logger.recordOutput("Autos/NotePositions/0", noteTranslation);
+    Logger.recordOutput("Autos/NotePositions/1", secondNoteTranslation);
+    Logger.recordOutput("Autos/NotePositions/2", thirdNoteTranslation);
+  }
+
+  public Command mobilityAuto() {
+    return new StraightDriveToPose(Units.feetToMeters(-6.75), 0, 0, drive);
+  }
+
+  public Command scoreOneNoteMobility() {
+    return Commands.sequence(oneNoteAuto().andThen(mobilityAuto()));
+  }
+
+  public Command oneNoteAuto() {
+    return orchestrator.fullAlignShootSpeaker();
```

Comment:
I'm worried that this could be faulty turnToSpeaker() or alignArmSpeaker() might not be accurate especially at close proximity to the speaker.

Should we have the first shot not do any alignment based on telemetry, and just shoot based on a fixed arm position - or at least have the option to do that?  @admlvntv @bryanduerk WDYT?

This applies to the first shot of all autos.